storage/range.go

```diff
@@ -124,17 +124,16 @@ var configDescriptors = [...]*configDescriptor{
 
 // tsCacheMethods specifies the set of methods which affect the
 // timestamp cache.
-var tsCacheMethods = [...]bool{
-	proto.Contains:                   true,
-	proto.Get:                        true,
-	proto.Put:                        true,
-	proto.ConditionalPut:             true,
-	proto.Increment:                  true,
-	proto.Scan:                       true,
-	proto.Delete:                     true,
-	proto.DeleteRange:                true,
-	proto.InternalResolveIntent:      true,
-	proto.InternalResolveIntentRange: true,
+var tsCacheMethods = [...]struct{}{
```

Comment:
using a `struct{}` map is the idiomatic way to model a set.  The bool is just extra information you don't need.